packages/server-wallet/src/models/channel.ts

```diff
@@ -138,6 +139,18 @@ export class Channel extends Model implements RequiredColumns {
     });
   }
 
+  get protocolState(): ChannelData {
```

Comment:
Will this eventually return transaction info as well?

Reply:
Yes! As well as adjudicator and asset holder info.